nemoguardrails/rails/llm/llmrails.py

```diff
@@ -619,19 +617,6 @@ async def generate_async(
         # Save the generation options in the current async context.
         generation_options_var.set(options)
 
-        if return_context:
-            warnings.warn(
-                "The `return_context` argument is deprecated and will be removed in 0.9.0. "
-                "Use `GenerationOptions.output_vars = True` instead.",
```

Comment:
Wow, we planned to remove it 5 point versions ago?

Reply:
Yes!

Pyramid of doom 🏔️ (llmrails.py)

Reply:
https://github.com/NVIDIA/NeMo-Guardrails/issues/1148